src/software/ai/hl/stp/tactic/patrol_tactic.h

```diff
@@ -25,12 +25,12 @@ class PatrolTactic : public Tactic
     std::string getName() const override;
 
     /**
-     * Updates the parameters for this PatrolTactic.
+     * Updates the world parameters for this PatrolTactic.
      *
      * @param orientation_at_patrol_points The orientation the robot
      * should have while patrolling
      */
-    void updateParams(Angle orientation_at_patrol_points);
+    void updateWorldParams(Angle orientation_at_patrol_points);
```

Comment:
Maybe it can be moved to the constructor? I can't think of a scenario when our patrol orientation would change dynamically.

Reply:
Addressed in c2ad653, made it a control parameter for now. @garethellis0 @jonathanlew do you think this can be moved to the constructor as @MathewMacDougall suggested?

Reply:
Yup, I agree with @MathewMacDougall here.

Reply:
Addressed in d8e9d966a2305392155236fb863a2d778e02c12a